test/policies/appliance_policy_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class AppliancePolicyTest < ActiveSupport::TestCase
+  def test_scope; end
```

Comment:
Your tests are doing nothing 🤔 
Of you **really don't** want to add them, let's get rid of them then :c

Reply:
I got rid of them for the time being. :/